### PR TITLE
v0.11.1: Add Refunder Contract

### PR DESCRIPTION
**Pull request by BrendanChou**

### CODE REVIEW

File: src/types.ts [hunk 1]
```diff
@@ -208,6 +209,14 @@ export interface SetExpiry extends AccountAction {
   expiryTime: Integer;
 }
 
+export interface Refund extends AccountAction {
+  receiverAccountOwner: address;
+  receiverAccountId: Integer;
+  refundMarketId: Integer;
+  otherMarketId: Integer;
```
Comment (AntonioJuliano) [inline src/types.ts]: What is this? Isn't this just unused?

Comment (BrendanChou) [reply src/types.ts]: Yes but we need to set it due to how AutoTrader is set up. And it cannot match refundMarketId. From a gas perspective, it should be the other market that was used in the trade.

Comment (AntonioJuliano) [reply src/types.ts]: Right, but we should just set it in solo.js and not have clients pass it in

Comment (AntonioJuliano) [reply src/types.ts]: or at least make it optional if you want to pass it in (if not passed autoset)

**Pull request merged**